org.eclipse.scanning.event.ui/src/org/eclipse/scanning/event/ui/view/StatusQueueView.java

```diff
@@ -830,7 +830,7 @@ public void dispose() {
 					for (Iterator it = retained.iterator(); it.hasNext();) {
 						StatusBean statusBean = (StatusBean) it.next();
 						if (statusBean.getUserName()==null) continue;
-						if (!showEntireQueue) {
+						if (hideOtherUsersResults) {
 							if (!userName.equals(statusBean.getUserName())) it.remove();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)